example/main.py

```diff
@@ -1,14 +1,14 @@
+import os
 from dotenv import load_dotenv
 load_dotenv()
-from database.connect import connect_to_db
-from database.write_data import write_data
-from models.get_llm_response import get_llm_response
-from datetime import datetime 
+from omnilog import get_llm_response, write_data, connect_to_db
```

Comment:
Is there a reason why `get_llm_response` is include as part of our package? What happens when we support other LLMs, not just openAI?